xds/internal/balancer/cdsbalancer/cdsbalancer_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build go1.12
```

Comment:
Did you want to keep this?

Reply:
This is the new build syntax - gofmt on my machine adds it.  It's harmless and will be the new standard going forward so I think it's fine.